focus/operations/benches/bench_sync.rs

```diff
@@ -16,7 +16,7 @@ pub fn bench_sync(c: &mut Criterion) {
     println!("Warming up with initial sync");
     focus_operations::sync::run(
         &repo_path,
-        focus_operations::sync::SyncMode::Normal,
+        focus_operations::sync::SyncMode::Incremental,
```

Comment:
Should we go ahead and add a benchmark for this version so we can make definitive statements later on about the speed of each strategy?

Reply:
Yeah, the queries are done in one shot. It doesn't lend itself to caching besides obviously for coarse caches -- the project cache can work with this fine.